omniscidb/QueryEngine/Optimization/AnnotateInternalFunctionsPass.h

```diff
@@ -37,7 +37,7 @@ class AnnotateInternalFunctionsPass
                               llvm::CGSCCAnalysisManager& AM,
                               llvm::LazyCallGraph& CG,
                               llvm::CGSCCUpdateResult& UR) {
-    bool updated_function_defs = false;
+    [[maybe_unused]] bool updated_function_defs = false;
```

Comment:
It looks like this is actually not used? @lmontigny

Reply:
I confirm, it's not used